todo/src/App.js

```diff
@@ -1,17 +1,60 @@
-import React from "react"
+import React, { useState } from "react"
 import TodoList from "./components/TodoList"
-import Counter from "./components/Counter"
+import "./App.css"
 
-const todosList = [
-	// { content: "go to the gym", isComplete: false },
-	// { content: "go to the mall", isComplete: false },
-	// { content: "finish assignment", isComplete: false },
-]
+const todosList = []
```

Comment:
You use this empty array as an initial state in a nested component. Wouldn't it be better to just initialize it with an empty array and delete this variable?